integrations/langchain/src/databricks_langchain/chat_models.py

```diff
@@ -307,8 +307,12 @@ def _convert_response_to_chat_result(self, response: Mapping[str, Any]) -> ChatR
             )
             for choice in response["choices"]
         ]
-        usage = response.get("usage", {})
-        return ChatResult(generations=generations, llm_output=usage)
+        llm_output = {
+            "token_usage": response.get("usage", {}),
```

Comment:
Can we rename this to usage so it doesn't change the API for existing users?

```suggestion
            "usage": response.get("usage", {}),
```

Reply:
I did as @bbqiu suggested, so `usage` remains intact

Reply:
I also created a new unit test, since the existing ones were not altered by this change